app/ide-desktop/lib/dashboard/src/modals/ProjectLogsModal.tsx

```diff
@@ -53,8 +55,20 @@ function ProjectLogsModalInternal(props: ProjectLogsModalInternalProps) {
   })
 
   return (
-    <pre className="relative overflow-auto whitespace-pre-wrap">
-      <code>{logsQuery.data}</code>
-    </pre>
+    <div className="flex flex-col">
+      <div className="flex h-row items-center gap-4 self-start rounded-full border-0.5 border-primary/20 px-[11px] text-primary/50">
+        <ariaComponents.Button
+          size="icon"
```

Comment:
nit: size="medium" should work well here

Reply:
replaced `size="icon"` with `size="medium"` everywhere, now that the size of icons is 16px for `size="medium"` i don't think there's any more reason for `size="icon"` to exist right?

Reply:
I think yes. We no longer need the `icon`(AFAIR we added it as a quick solution for icon buttons in the dashboard redesign PR). Since then I spent some time aligning the buttons inside the grid so we no longer have issues with the icon sizes.